aws_codeseeder/commands/_seedkit_commands.py

```diff
@@ -55,6 +55,8 @@ def deploy_seedkit(
     subnet_ids: Optional[List[str]] = None,
     security_group_ids: Optional[List[str]] = None,
     permissions_boundary_arn: Optional[str] = None,
+    synthesize: bool = False,
```

Comment:
I feel like this is misnamed -- it always synthesizes, maybe `synthesize_only` is a better name?

Reply:
Kept the naming consistent with equivalent Seed-Farmer command as so far those are mostly used together.